clar2wasm/tests/wasm-generation/bitwise.rs

```diff
@@ -3,11 +3,74 @@ use proptest::proptest;
 
 use crate::{int, uint};
 
+const ONE_OP: [&str; 1] = ["bit-not"];
+const TWO_OPS: [&str; 2] = ["bit-shift-left", "bit-shift-right"];
+const MULTI_OPS: [&str; 3] = ["bit-and", "bit-or", "bit-xor"];
+
```

Comment:
There are no tests for `xor`, which is the same as `bit-xor` except that it only supports two expressions.

Reply:
Added tests for `xor`